kinsumer.go

```diff
@@ -483,6 +484,51 @@ func (k *Kinsumer) CreateRequiredTables() error {
 	return g.Wait()
 }
 
+func (k *Kinsumer) ResetCheckpoints(sess *session.Session) error {
+	entries, err := k.dynamodb.Scan(&dynamodb.ScanInput{TableName: aws.String(k.checkpointTableName)})
+	if err != nil {
+		return err
+	}
+
+	for _, entry := range entries.Items {
+		shard := entry["Shard"].S
+
+		if shard == nil {
+			return fmt.Errorf("found %v for Shard entry", entry["Shard"].S)
+		}
+
+		err := k.updateItem(*shard)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (k *Kinsumer) updateItem(id string) error {
+	condition := expression.Name("Shard").Equal(expression.Value(id))
+	var update = expression.UpdateBuilder{}
+	exp, err := expression.NewBuilder().WithCondition(condition).WithUpdate(
+		update.Set(expression.Name("SequenceNumber"), expression.Value("LATEST")),
+	).Build()
+	if err != nil {
+		return err
+	}
+	_, err = k.dynamodb.UpdateItem(&dynamodb.UpdateItemInput{
```

Comment:
For a large table this could get slow. Especially since you are reusing the same expression (just difference condition) for each update. Also there is no `batchUpdateItem` so maybe we should consider going wide. Probably fine though.